responsive-web-design/css-typography/index12.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Nutrition Label</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800" rel="stylesheet">
+  </head>
+  <style>
+    body {
+      font-family: 'Open Sans', sans-serif;
+      font-size: 16px;
+    }
+
+    .label {
+      border: 2px solid black;
+      width: 270px;
+    }
+  </style>
+  <body>
+    <div class="label">
+      <h1>Nutrition Facts</h1>
+      <p>8 servings per container</p>
+      <p>Serving size 2/3 cup (55g)</p>
+    </div>
+  </body>
+</html>
+
+<!-- Now center the label and add a top and bottom margin with the shorthand `margin: 20px auto;` declaration. -->
```

Comment:
Add more context here describing the `margin` rule. You don't have to be as complex as MDN's breakdown but at least explain the 2 main variants of `margin: topbottom leftright` and `margin: top right bottom left` as well as maybe describing the `auto` keyword.